pandas/core/internals.py

```diff
@@ -861,6 +861,8 @@ def _is_empty_indexer(indexer):
 
         # set
         else:
+            # Work around GH 6168 to support numpy 1.7 and 1.8
+            indexer = getattr(indexer, 'values', indexer)
```

Comment:
you can be explcit and use _np_version_under1p19 (then this is easier to remove later). alternatively the ``np.asarray`` may work.